llvm/test/CodeGen/NVPTX/surf-tex.py

```diff
@@ -1,12 +1,12 @@
 # RUN: %python %s --target=cuda --tests=suld,sust,tex,tld4 --gen-list=%t.list > %t-cuda.ll
-# RUN: llc -mcpu=sm_20 %t-cuda.ll -verify-machineinstrs -o - | FileCheck %t-cuda.ll
```

Comment:
We should still have the CPU set explicitly. `sm_60` would probably be the reasonable oldest non-deprecated arch.

Reply:
Sounds good, fixed.